google-cloud-video-transcoder/samples/create_job_with_concatenated_inputs.rb

```diff
@@ -0,0 +1,157 @@
+# Copyright 2021 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def create_job_with_concatenated_inputs project_id:, location:, input1_uri:, start_time_input1:, end_time_input1:, input2_uri:, start_time_input2:, end_time_input2:, output_uri:
+  # [START transcoder_create_job_with_concatenated_inputs]
+  # project_id  = "YOUR-GOOGLE-CLOUD-PROJECT"  # (e.g. "my-project")
+  # location    = "YOUR-JOB-LOCATION"  # (e.g. "us-central1")
+  # input1_uri   = "YOUR-GCS-INPUT-VIDEO1"  # (e.g. "gs://my-bucket/my-video-file1")
+  # start_time_input1   = 0 # Start time in fractional seconds relative to the first input video timeline
+  # end_time_input1   = 8.125 # End time in fractional seconds relative to the first input video timeline
+  # input2_uri   = "YOUR-GCS-INPUT-VIDEO2"  # (e.g. "gs://my-bucket/my-video-file2")
+  # start_time_input2   = 3.5 # Start time in fractional seconds relative to the second input video timeline
+  # end_time_input2   = 15 # End time in fractional seconds relative to the second input video timeline
+  # output_uri  = "YOUR-GCS-OUTPUT-FOLDER/"  # (e.g. "gs://my-bucket/my-output-folder/")
```

Comment:
These look really "messy". Can you clean it up a bit? e.g.:

* Some of the lines are pretty long. Consider wrapping at 80 cols.
* Spacing is highly inconsistent. Sometimes you have 2 spaces between the variable name and `=`, sometimes 3. It should be consistently 1, unless you're trying to align the `=` operators on the same column. Similarly, sometimes you have 2 spaces before the comment, sometimes 1. It would be really good to align those vertically if possible, but if not, you should at least have consistent spacing.

Reply:
Done